Sources/JavaScriptKit/BasicObjects/JSError.swift

```diff
@@ -0,0 +1,20 @@
+public final class JSError {
+    private let ref: JSObject
+    private static let constructor = JSObject.global.Error.function!
+
+    public init(message: String) {
+        ref = Self.constructor.new([message])
+    }
+
+    public var message: String {
+        ref.message.string!
+    }
+
+    public var name: String {
+        ref.name.string!
+    }
+}
```

Comment:
Although it’s not standard, all engines that I know of implement `error.stack` to provide a stack trace as a string. It might be useful to access it.